job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/StepContext.groovy

```diff
@@ -651,6 +651,15 @@ class StepContext implements Context {
                             configs('class': 'java.util.Collections$EmptyList')
                         }
                     }
+
+                    // Apply overrides
+                    if (jobInPhase.configureClosure) {
+                        jobInPhase.configureClosure.resolveStrategy = Closure.DELEGATE_FIRST
```

Comment:
Is there a reason why you set the resolveStrategy? Most configure blocks use the default strategy, except for the extendedEmail configure block. I would like to use the same strategy everywhere to avoid surprises.